html-forms/html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label>Email
+          <div>
+            <input type="text" name="email">
+          </div>
+        </label>
+      </div>
+      <div>
+        Why is coding cool? <br  /><select name="reasons">
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <br  />
```

Comment:
last thing is to remove all the `br` tags. We only use those for text formatting, not page formatting